README.md

```diff
@@ -1,22 +1,37 @@
 FULLY AUTOMATED REDISTICTING SOFTWARE
 written in java, open source.
 
-CONTRIBUTION GUIDELINES:
```

Comment:
like it - maybe at the end add how to make an executable jar, and then how
to launch that? (a jar is really just a .zip file)  makes it a little more
transportable.

i don't currently have a manifest file in the source but i think that might
make launching simpler - might not need to specify the main class, then.

On Mon, Jan 18, 2016 at 10:41 PM, Carl Schroedl <notifications@github.com>
wrote:

> In README.md
> <https://github.com/happyjack27/autoredistrict/pull/4#discussion_r50072236>
> :
>
> > @@ -1,22 +1,37 @@
> >  FULLY AUTOMATED REDISTICTING SOFTWARE
> >  written in java, open source.
> >
> > -CONTRIBUTION GUIDELINES:
>
> I moved the contrib guidelines further down so that the "HOW TO RUN"
> section would be first.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/happyjack27/autoredistrict/pull/4/files#r50072236>.
>